common/arrow/Cargo.toml

```
@@ -20,7 +20,7 @@ simd = ["arrow/simd"]
 # Workspace dependencies
 
 # Github dependencies
-arrow = { package = "arrow2", git = "https://github.com/datafuse-extras/arrow2", default-features = false, rev = "6ec9cf"}
+arrow = { package = "arrow2", git = "https://github.com/datafuse-extras/arrow2", default-features = false, rev = "89921d33bdb4becf8e9197fdc8392818862357cd"}
```

```suggestion
arrow = { package = "arrow2", git = "https://github.com/datafuse-extras/arrow2", default-features = false, rev = "565cfae"}
```

We may consider using the latest commit. There are two bug fixes in between.